app/static/src/app/store/graphs/getters.ts

```diff
@@ -3,15 +3,21 @@ import { GraphsState } from "./state";
 import { AppState } from "../appState/state";
 
 export enum GraphsGetter {
-    allSelectedVariables = "allSelectedVariables"
+    allSelectedVariables = "allSelectedVariables",
+    hiddenVariables = "hiddenVariables" // variable which are not selected in any graph
 }
 
 export interface GraphsGetters {
     [GraphsGetter.allSelectedVariables]: Getter<GraphsState, AppState>;
+    [GraphsGetter.hiddenVariables]: Getter<GraphsState, AppState>;
 }
 
 export const getters: GraphsGetters & GetterTree<GraphsState, AppState> = {
     [GraphsGetter.allSelectedVariables]: (state: GraphsState): string[] => {
-        return state.config.flatMap((c) => c.selectedVariables);
+        return state.config.flatMap((c) => c.selectedVariables); // TODO: dedupe, in mrc-5443
+    },
+    [GraphsGetter.hiddenVariables]: (_, graphsGetters, rootState): string[] => {
```

Comment:
explicit param types would be great

Reply:
This is a little bit gruesome in the case of the getters parameter, because you don't call the getter explicitly, so you can't use `GetterTree` which is an array of methods (which in any case have result type `any` which isn't what we want...) I'll try just declaring a separate interface for the getters values as available from another getter - see if you think that works!